src/WebWindow.Native/WebWindow.Windows.cpp

```diff
@@ -225,7 +225,19 @@ void WebWindow::AttachWebView()
 						Settings->put_IsScriptEnabled(TRUE);
 						Settings->put_AreDefaultScriptDialogsEnabled(TRUE);
 						Settings->put_IsWebMessageEnabled(TRUE);
-
+                        // Add a navigation change event handler
+						EventRegistrationToken token;
+                        _webviewWindow->add_NavigationStarting(Callback<IWebView2NavigationStartingEventHandler>(
+                            [this](IWebView2WebView* webview, IWebView2NavigationStartingEventArgs * args) -> HRESULT 
+                            {
+                                PWSTR uri;
+                                args->get_Uri(&uri);
+                                _uriChangeCallback(uri);
+                                CoTaskMemFree(uri);
```

Comment:
This CoTaskMemFree comes from https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2/gettingstarted#step-4---navigation-events

Direction needed on whether it should be removed or not.

Reply:
How about using `wil::unique_cotaskmem_string`? It is just like `unique_ptr` and `CoTaskMemFree` will be called automatically.

Reply:
Updated!